### PR TITLE
Added Three.js template

### PR DESCRIPTION
**Pull request by marioecg**
After reading [Jaume's port of Makio's 3D animation](https://twitter.com/thespite/status/1383413661687943168) I think it was good to make a Three.js template for anyone to use.

### CODE REVIEW

File: templates/three-template/script.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+console.log('NFT created by', creator);
+console.log('NFT viewed by', viewer);
+
+class Sketch {
+  constructor() {
+    this.renderer = new WebGLRenderer({ 
+      antialias: true, 
+      alpha: true, 
+    });
+    this.renderer.setPixelRatio(Math.min(window.devicePixelRatio, 1.5));
```
Comment (andrevenancio) [inline templates/three-template/script.js]: I normally use 2 instead of 1.5. Never tried it like this! have you ever tested it on a retina display?

**Review by andrevenancio — APPROVED** [review]
Looks great! thanks @marioecg

File: templates/three-template/script.js (hunk 1 above)
Comment (marioecg) [reply templates/three-template/script.js]: I can change it, haven't tested it either, grabbed the number from the reference repo.

**Comment by mrdoob** [comment]
Shouldn't the folder be named `html-three-template`? 😇https://github.com/hicetnunc2000/hicetnunc/tree/main/templates

**Comment by marioecg** [comment]
Right @mrdoob! Requested it here #545

**Pull request merged**